src/checkFen.js

```diff
@@ -2,24 +2,56 @@ function checkFen(fenString) {
     var isValid = true;
     var fenArray = [];
 
+    //check if fenString is null (When cancel is pressed during prompt)
+    if (fenString == null) {
+        return false;
+    }
+
     //check if fenString is valid with respect to the number of pieces
     fenArray = getCapturedPieces(1, fenString);
 
+    const piece = ["King", "Queen", "Rook", "Knight", "Bishop", "Pawn"];
+
     for (let i = 0; i < fenArray.length; i++) {
         if (fenArray[i] < 0) {
             isValid = false;
+            if (i < 6) {
+                customAlert("Invalid FEN position\nNumber of black " + piece[i] + "'s is not valid");
```

Comment:
Please remove 's as it is grammatically incorrect. Thank you.

Reply:
For example. 
- number of black rooks is correct
- number of black rook's is incorrect